src/Controls/samples/Controls.Sample.UITests/Issues/Issue18242.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue18242"
+             Title="Issue 18242">
+    <VerticalStackLayout
+        Padding="12">
+        <Label
+            AutomationId="WaitForStubControl"
+            Text="1. The image must adapt to the size of the Button and be in the position determined by the ContentLayout property."/>
+        <Button
```

Comment:
Should we add a test for when there is _no_ size set on the button to make sure that code path is still working? If I don't set a button size, then the image affects the button size. Might need to add a test where Width is set and not height.

Reply:
Good idea. Updated the sample:
![image](https://github.com/dotnet/maui/assets/6755973/f211ea94-63bf-49aa-bc05-96e440855ed3)

This requires to update the golden test reference snapshots (next build will fail).

Reply:
Updated reference snapshots.